tests/test_fastadir.py

```diff
@@ -37,3 +37,25 @@ def test_write_reread():
     import logging
     logging.basicConfig(level=logging.DEBUG)
     test_write_reread()
+
+
+def test_schema_version():
+    tmpdir = tempfile.mkdtemp(prefix="seqrepo_pytest_")
+    orig_schema_version = FastaDir.schema_version
+
+    with pytest.raises(RuntimeError):
+        FastaDir.schema_version = lambda x: 2
```

Comment:
When FastDir.schema_version is updated (i.e., 2 *is* the version), this test will break by no longer raising an exception, right?

How about setting to `-1` instead?